### ISSUE TITLE
Webapp:Cannot add the same webapp restriction rule to scm site when rules is already in the main site

### ISSUE THREAD

**Issue by kmadof** [Azure/azure-cli#15198]
**Describe the bug**
When I add rule to the main site and then try to add rule to scm site with the same ip address I got this error:

> IP address 4.3.2.1/32 already exists.Cannot add duplicate IP address values.

However, when I first add tule to scm site and then to main site all is fine.

**To Reproduce**

Run command in this order:
```
az webapp config access-restriction add --resource-group 'TheCodeManual' --name 'angular-webapp-demo' --action Allow --priority 20 --ip-address '4.3.2.1/32' --rule-name 'Test1' --description 'Test1'

az webapp config access-restriction add --resource-group 'TheCodeManual' --name 'angular-webapp-demo' --action Allow --priority 21 --ip-address '4.3.2.1/32' --rule-name 'Test2' --description 'Test2' --scm-site $true
```
**Expected behavior**
I know I can use `az webapp config access-restriction set --use-same-restrictions-for-scm-site ` but this behavior is counterintuitive. I expect to be able to add rule to scm in that way.

**Environment summary**
I tested this on Azure Cloud shell

```
azure-cli                         2.11.1

core                              2.11.1
telemetry                          1.0.5 *

Extensions:
ai-examples                        0.2.3
```

**Comment by yonzhan** [comment]
webapp

**Comment by JacobStruiksma** [comment]
The same Issue as #14862 But that one is not been resolve yet.

**Comment by Mjiduser** [comment]
Same issue here, but figured it out by adding first scm site rules  and then the main app site.

**Comment by btardif** [comment]
This is fixed by #15784
